src/framework/pages/admin/resources/ResourceAssociationsPage.java

```diff
@@ -1,5 +0,0 @@
-package framework.pages.admin.resources;
```

Comment:
Be careful this is incorrect, please check your changes and do the process (commit -> pull -> commit - > push) again!!!!